environment.yml

```diff
@@ -0,0 +1,25 @@
+name: ibis-dask-dev
+channels:
+  - conda-forge
+dependencies:
+    # Hard dependencies
+    - multipledispatch>=0.6.0
+    - numpy>=1.19
+    - pandas>=1.1
```

Comment:
ibis provides numpy, pandas... I don't see much value on adding them explicitly, unless you're doing it to speed up the conda solver (I'd add a comment if that's the case)

Reply:
Since the dask backed uses them explicitly, I wanted to also keep them around here. It does also make the solver faster.

Reply:
That makes sense, didn't think about it, better to have them here.